src/docs/cookbook/testing/integration/index.md

```diff
@@ -2,4 +2,10 @@
 title: Integration
 ---
 
+{{site.alert.note}}
+The integration_test package is now the recommended way to write integration
```

Comment:
Indent note content by two spaces on each line. (By the way, Andrew is quite the expert on the code snippets now..). Also, code font `integration_test1` here and everywhere, except headers, alt text, etc.
Also, we use sentence case for titles, so it should be "Integration testing" with a lowercase t.